### PR TITLE
Fix success_count not reset issue

### PR DESCRIPTION
**Pull request by whikwon**
#58

### CODE REVIEW

File: scripts/envs/open_manipulator/ros_interface.py [hunk 1]
```diff
@@ -306,7 +306,6 @@ def check_for_success(self):
             self.success_count += 1
             if self.success_count == self.cfg["SUCCESS_COUNT"]:
                 print ("Current episode succeeded")
-                self.success_count = 0
```
Comment (seungjaeryanlee) [inline scripts/envs/open_manipulator/ros_interface.py:309]: Why is this line deleted? To I am unfamiliar with `self.success_count` and `self.cfg["SUCCESS_COUNT"]`

File: scripts/envs/open_manipulator/ros_interface.py [hunk 2]
```diff
@@ -388,6 +387,7 @@ def reset_gazebo_world(self, block_pose=None):
 
         # self.delete_target_block()
         self.init_robot_pose()
+        self.success_count = 0
```
Comment (seungjaeryanlee) [inline scripts/envs/open_manipulator/ros_interface.py:390]: This line looks good. We want to reset it when reset is called.

**Review by seungjaeryanlee — COMMENTED** [review]
Looks good: I am just unfamiliar with success_count so I would appreciate some explanation of what "success" is. Why it is not a boolean value (success vs failure) and instead a counter?

File: scripts/envs/open_manipulator/ros_interface.py (hunk 1 above)
Comment (whikwon) [reply scripts/envs/open_manipulator/ros_interface.py:309]: `self.cfg["SUCCESS_COUNT"]` means the number of steps stayed in boundary to goal.

Comment (whikwon) [reply scripts/envs/open_manipulator/ros_interface.py:309]: As `self.success_count` reset to 0 when `check_for_success` return `True`, I deleted it .

Comment (seungjaeryanlee) [reply scripts/envs/open_manipulator/ros_interface.py:309]: I see. So, in [open_manipulator_reacher_env.py](https://github.com/kairproject/kair_algorithms_draft/blob/c1f348775e41c71d0de1bc6699f7a1732ee0bed4/scripts/envs/open_manipulator/open_manipulator_reacher_env.py#L81), if `check_for_success` is True, then `self.done` is set to true. It doesn't explicitly reset it to 0, but since the episode is complete, the next call would be `reset()`, which calls `reset_gazebo_world()` and resets the counter to 0.

Is this correct?

Comment (whikwon) [reply scripts/envs/open_manipulator/ros_interface.py:309]: Yes, that's correct.

**Review by seungjaeryanlee — APPROVED** [review]
